src/tests/Common/test_dependencies/test_dependencies.csproj

```diff
@@ -10,6 +10,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Diagnostics.Tools.RuntimeClient" Version="$(MicrosoftDiagnosticsToolsRuntimeClientVersion)" />
+    <!-- Required by the GitHub_25468 test project. -->
+    <PackageReference Include="System.Drawing.Common" Version="$(SystemDrawingCommonVersion)" />
```

Comment:
This appears to be a JIT regression test https://github.com/dotnet/runtime/commit/92a49fbf421ed0df730956b6d3404597189ee97c.  Issue https://github.com/dotnet/runtime/issues/12993.  Given we're likely not even exercising the libgdiplus scenario anymore I wonder how valuable this test is.  Maybe file a follow up to clean this up.